spec/models/concerns/inventory_legacy_support_spec.rb

```diff
@@ -0,0 +1,399 @@
+require 'rails_helper'
+
+describe InventoryLegacySupport do
+  let(:location) { create(:location) }
+  let(:other_location) { create(:location) }
+  let(:user) { create(:user) }
+  let(:package) { create(:package) }
+  let(:other_package) { create(:package) }
+  let(:packages_location) { create(:packages_location, quantity: quantity, location: location, package: package) }
+  let(:computer) { PackagesInventory::Computer }
+
+  before { User.current_user = user }
+
+  after do
```

Comment:
Just to make things really clear
```suggestion
  after(:each) do
```